sklearn/ensemble/weight_boosting.py

```diff
@@ -552,6 +559,8 @@ def _boost_discrete(self, iboost, X, y, sample_weight, random_state):
         # Error fraction
         estimator_error = np.mean(
             np.average(incorrect, weights=sample_weight, axis=0))
+        if math.isnan(estimator_error):
+            return None,None,estimator_error
```

Comment:
idem

Reply:
what does "idem" mean? Sorry if the question is lame.